lib/rules/no-duplicate-selectors/__tests__/index.mjs

```diff
@@ -521,5 +521,8 @@ testRule({
 			code: 'a { b, .@{var} {} b {} }',
 			description: 'Non standard Less nested interpolation (3',
 		},
+		{
+			code: '.foo() { .bar > & { color: #000; } &.readonly { color: #fff; } }',
```

Comment:
I suppose the mixin is being ignored in its entirety.
i.e. if there are duplicates in `.foo() { /* here */ }`, they won't be detected